src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -802,22 +802,28 @@ def max_timestamp(aggregate_filter, org_id, use_case_id, alias=None):
     return timestamp_column_snql("maxIf", aggregate_filter, org_id, use_case_id, alias)
 
 
-def on_demand_failure_rate_snql_factory(aggregate_filter, org_id, use_case_id, alias=None):
+def total_count(aggregate_filter: Function) -> Function:
+    return Function("sumIf", [Column("value"), aggregate_filter])
```

Comment:
We use this function in a couple of places, thus, we can use this helper function.